proposals/lip-0048.md

```diff
@@ -241,10 +252,11 @@ def verify(trs: Transaction) -> None:
     b = block including trs
     h = b.header.height
     if h < MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE:
-	    # Set MIN_FEE_PER_BYTE = 0 for the first MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE blocks.
-        MIN_FEE_PER_BYTE = 0
-    trsSize = length(encodeTransaction(trs))
-    minFee = MIN_FEE_PER_BYTE * trsSize
+	    # Set minFee = 0 for the first MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE blocks.
```

Comment:
indentation is off.

Reply:
I don't know what is happening with my VSC, I'll commit from this suggestion:
```suggestion
    # Set minFee = 0 for the first MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE blocks.
```

Reply:
Second try:
```suggestion
        # Set minFee = 0 for the first MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE blocks.
```